src/constructs/iam/policies/ses.ts

```diff
@@ -1,22 +1,14 @@
-import { Effect, PolicyStatement } from "@aws-cdk/aws-iam";
 import type { GuStack } from "../../core";
 import { GuGuardianEmailSenderParameter } from "../../core";
-import type { GuPolicyProps } from "./base-policy";
-import { GuPolicy } from "./base-policy";
-
-export class GuSESSenderPolicy extends GuPolicy {
-  constructor(scope: GuStack, id: string = "GuSESSenderPolicy", props?: GuPolicyProps) {
-    super(scope, id, { ...props });
+import { GuAllowPolicy } from "./base-policy";
 
+export class GuSESSenderPolicy extends GuAllowPolicy {
```

Comment:
Extending `GuAllowPolicy` yields a simpler implementation of this policy 🎉 .

Reply:
💯